client/src/models/populate_results.js

```diff
@@ -238,6 +249,13 @@ function extract_flat_data_from_results_hierarchies(hierarchical_response_data){
     indicators = [],
     pi_dr_links = [];
 
+  const process_indicator = (indicator) => {
+    indicator.target_year = _.isEmpty(indicator.target_year) ? null : parseInt(indicator.target_year);
+    indicator.target_month = _.isEmpty(indicator.target_month) ? null : parseInt(indicator.target_month);
```

Comment:
Nitpicky, and fact check me on this, but I think empty target year values come out of the API as a nulls already.

Reply:
Less nitpicky reason to delete this, I think this is the only part of `crawl_hierachy_level` that mutates the data passed to it. Possibly irrelevant, because I'm not sure the original object is used again after this, but still something to generally avoid.

Reply:
wontfix in favour of #321 😝